wiki/Beatmap_Discussion/en.md

```diff
@@ -1,225 +1,183 @@
+When a beatmap is [submitted](/wiki/Submission), a discussion page is automatically created alongside the beatmap info page. When the beatmap is updated, the discussion page will be updated along with it. The discussion page can be reached by clicking on the `Discussion` button on the info page. <!--TODO: insert link to beatmap info page article, when that article exists-->
 
-### New beatmaps
+The beatmap discussion page consists of a few elements (from top-to-bottom):
 
-New beatmaps submitted via the Beatmap Submission System will use Beatmap Discussion.
+- Mode selection
+- Beatmap header
+- Difficulty menu
+- Sorting options
+- Visual timeline
+- Beatmap status
+- Modding tabs
+- Submission field
+- Discussions
 
-### Old beatmaps
+You can follow along with our images from [this beatmap](https://osu.ppy.sh/beatmapsets/924551/discussion/) to get familiar with Beatmap Discussions or try opening a discussion page yourself!
 
-A staff member has to move the beatmap manually to use Beatmap Discussion. [Ephemeral](https://osu.ppy.sh/users/102335) is currently in charge of transferring beatmapsets. All existing beatmaps in the WIP and pending statuses that do not have any mod posts in the forum have been converted to use Beatmap Discussion.
+## Quick-start: Using Beatmap Discussions
 
-### Returning to the old modding system
+It's important to keep a few things in mind to use the Beatmap Discussion pages effectively:
 
-It is no longer possible to return to the old modding system. The only beatmaps that may use the old system are those with mod posts. If beatmaps already use Beatmap Discussion, the forum thread will not have a reply button, instead, it will replaced with a button that redirects to the Beatmap Discussion panel.
-
-## Accessing the Beatmap Discussion page
-
-### Through new.ppy.sh:
+- Choose the correct mode and difficulty you wish to mod.
+- Select the proper section you are submitting your mod to.
+- **Only post one issue at a time.** Do not submit multiple issues in a single post.
+- Check to see if the issue has already been addressed when the similar post warning pops up.
+- Do you like the beatmap? Give your hype to help the beatmap move closer to nomination.
 
-Click on the `Discussion` button in the beatmap info page.
+## Mode selection
 
-![](img/info.jpg "Beatmap info page")
+![](img/mode-selection.jpg "Mode selection.")
 
-### Through osu.ppy.sh:
+Mode selection changes the game mode between the different modes: [osu!standard](/wiki/Game_Modes/osu!), [osu!taiko](/wiki/Game_Modes/osu!taiko), [osu!catch](/wiki/Game_Modes/osu!catch), and [osu!mania](/wiki/Game_Modes/osu!mania). It is only possible to select a game mode if the beatmapset has a difficulty of that game mode. This feature is mainly used for beatmaps which contain difficulties of multiple game modes. For normal beatmaps this remains untouched, since the discussion page automatically switches to the current game mode.
```

Comment:
```suggestion
Mode selection changes the [game mode](/wiki/Game_mode) you are modding. It is only possible to select a game mode if the beatmapset has a difficulty of that game mode. This is only used with beatmaps that contain multiple game modes; otherwise, it will default to the mapped game mode.
```